src/main/java/com/archisacademy/jobportal/services/impl/CommentServiceImpl.java

```diff
@@ -0,0 +1,121 @@
+package com.archisacademy.jobportal.services.impl;
+
+import com.archisacademy.jobportal.dto.CommentDto;
+import com.archisacademy.jobportal.loggers.MainLogger;
+import com.archisacademy.jobportal.loggers.messages.CommentMessage;
+import com.archisacademy.jobportal.loggers.messages.UserMessage;
+import com.archisacademy.jobportal.mapper.CommentMapper;
+import com.archisacademy.jobportal.model.Comment;
+import com.archisacademy.jobportal.repositories.CommentRepository;
+import com.archisacademy.jobportal.repositories.PostRepository;
+import com.archisacademy.jobportal.repositories.UserRepository;
+import com.archisacademy.jobportal.services.CommentService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CommentServiceImpl implements CommentService {
+    private final CommentRepository commentRepository;
+    private final PostRepository postRepository;
+    private final UserRepository userRepository;
+    private final CommentMapper commentMapper;
+    private final static MainLogger LOGGER = new MainLogger(CommentServiceImpl.class);
+
+    public CommentServiceImpl(CommentRepository commentRepository, PostRepository postRepository, UserRepository userRepository, CommentMapper commentMapper) {
+        this.commentRepository = commentRepository;
+        this.postRepository = postRepository;
+        this.userRepository = userRepository;
+        this.commentMapper = commentMapper;
+    }
+
+    @Override
+    @Transactional
+    public String createComment(CommentDto commentDto) {
+
+        userRepository.findByUuid(commentDto.getUserUuid())
```

Comment:
you should store the fetched user inside an object. right now it does not make since